core/builtin.py

```diff
@@ -1088,7 +1099,7 @@ def Umask(argv):
     # NOTE: dash disables interrupts around the two umask() calls, but that
     # shouldn't be a concern for us.  Signal handlers won't call umask().
     mask = os.umask(0)
-    os.umask(mask)  # 
```

Comment:
Sorry about the whitespace diff lines; I auto-indented the whole file because I inadvertently started out using 4-width indents in the new code.